src/dropbox.ml

```diff
@@ -183,6 +185,8 @@ module type S = sig
       contents: metadata list;
     }
 
+  type search = metadata list
```

Comment:
I think exporting this type only obscure the interface — one more indirection to remember...